src/content/overview/architecture/_index.md

```diff
@@ -11,7 +11,57 @@ user_questions:
   - What is the architecture of the Giant Swarm cloud-native developer platform?
 owner:
   - https://github.com/orgs/giantswarm/teams/team-planeteers
-last_review_date: 2024-04-26
+last_review_date: 2024-11-04
 ---
 
-## Platform API {#platform-api}
```

Comment:
@JosephSalisbury, I added the diagram. I added an intro, I explained the layers; and now my idea is to dissect all the boxes. I want to add a paragraph by box to avoid making it super long and point to the right section to read more. WDYT?